### ISSUE TITLE
Manage Package for Solutions shows error while browsing packages from an internal JFrog NuGet artifactory

### ISSUE THREAD

**Issue by buddhashrestha** [NuGet/Home#8856]
NuGet product used (NuGet.exe | VS UI | Package Manager Console):

NuGet version (x.x.x.xxx): 5.3.1.6268

dotnet.exe --version (if appropriate): 3.0.100

VS version (if appropriate): 2017

OS version (i.e. win10 v1607 (14393.321)): win10

Worked before? If so, with which NuGet version: No

Prerequisite: Downloaded latest nuget.exe 

1. Created and pushed a NuGet package to an internal JFrog artifactory. Everything works fine and I can view my package at the artifactory. i.e. nuget push <packageName> is successful.

2. When I try to connect the artifactory from Visual studio ( Tools > NuGet Package Manager > Manage Packages for Solution), It just shows me an error message 
![image](https://user-images.githubusercontent.com/10449218/69584225-88527000-0f91-11ea-9024-27d6420f735b.png)
I believe, if it doesnot find any apikey, then it should allow me to enter the apikey. But it does nothing.

3. I set up my api key from package manager console from the Visual Studio 17, with the following command: 
nuget setapikey <USERNAME>:<PASSWORD> -Source Artifactory

4. I try to list all the packages in my artifactory using command : nuget list 
But it hangs for a while and gives me following error
![image](https://user-images.githubusercontent.com/10449218/69584535-3e1dbe80-0f92-11ea-9601-4cdae1716d64.png)
Here, it first says Please provide credentials for ... for there is no space to enter the username and password. It does nothing and it waits for the timeout.

5. As an alternative, I perform steps 3 and 4 from cmd, it works and shows me all the packages. So, I'm assuming something is wrong while using it with Visual Studio 17. 
...

**Comment by kartheekp-ms** [comment]
@buddhashrestha - Similar issue is tracked here #8801. Please click on `Show errors in output` button and let us know if the error message is more verbose than the message displayed on  `nuget.exe` CLI.
![image](https://user-images.githubusercontent.com/10449218/69584225-88527000-0f91-11ea-9024-27d6420f735b.png)

**Comment by buddhashrestha** [comment]
Hi @kartheekp-ms , the Show errors in output button, doesn't do any thing i.e. the output console is empty. My assumption was, if there is an apikey missing/error, then it should let me reenter it.

**Comment by buddhashrestha** [comment]
Hi @kartheekp-ms , thanks for the reply. Here is the image of the error message on the output.
![image](https://user-images.githubusercontent.com/10449218/69678622-f151ea80-105a-11ea-93a7-bb59624f2362.png)
Here it says 403 whereas from the cmd, I can list my packages.

So, I guess we need to check from where is Visual Studio getting my the config from and see if the apikey is the updated there or not.

**Comment by kartheekp-ms** [comment]
The API key is only used for pushing packages. It is not used for authenticating to a private feed.`list` command hits different end points when it is invoked from `command line interface` and `VS`.
Please refer to this [doc](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesourcecredentials) to setup usernames and passwords for sources.

**Comment by buddhashrestha** [comment]
Thank you. I deleted the old package source and added a new one from the package manager console in VS with the username and password (nuget sources Add -Name Artifactory -Source https://xyz.jfrog.io/xyz/api/nuget/nuget-local-test -username <USERNAME> -password <PASSWORD>) and then restarted the VS. This sequence worked for me. Also, I found out that my NuGet.config was in C:\Users\<You User Name>\AppData\Roaming\NuGet directory.
